### ISSUE TITLE
Version 0.0.4 is broken with Test::More::YAMLScript 0.1.3

### ISSUE THREAD

**Issue by andk** [ingydotnet/yamltest#2]
Sample fail report: http://www.cpantesters.org/cpan/report/811b7cd4-0dab-11ee-a47a-d2514ed2e7f5

Diagnostics:

``` text
Can't locate object method "new" via package "YAMLScript" at blib/script/yamltest line 12.
```

Somewhere between Test::More::YAMLScript 0.0.5 and 0.1.3 the breakage occurred, I do not have sufficient reports to tell the exact breaking event.

**Comment by andk** [comment]
@ingydotnet , do you need further information? Since 2023-06-14 nobody has sent a pass for YAMLTest. Here is a more recent fail report: http://www.cpantesters.org/cpan/report/06ddbe62-17d0-11ee-8ddf-ca5fc877322c

**Comment by ingydotnet** [comment]
I'm planning on writing on that module this weekend. Thanks!

**Comment by ingydotnet** [comment]
Just released 0.1.0 which should fix things. Closing for now. Thanks!
